tests/util/test_config.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+from typing import Any, Dict
+
+import pytest
+
+from chia.server.outbound_message import NodeType
+from chia.util.config import get_unresolved_peer_infos
+
+
+@pytest.mark.asyncio
+async def test_get_unresolved_peer_infos_with_multiple_farmer_peers() -> None:
+    service_config = {
+        "farmer_peers": [
+            {
+                "host": "127.0.0.1",
+                "port": 8447,
+            },
+            {
+                "host": "my.farmer.tld",
+                "port": 18447,
+            },
+        ],
+    }
+
+    peer_infos = get_unresolved_peer_infos(service_config, NodeType.FARMER)
+
+    assert len(peer_infos) == 2
+    assert peer_infos[0].host == "127.0.0.1"
+    assert peer_infos[0].port == 8447
+    assert peer_infos[1].host == "my.farmer.tld"
+    assert peer_infos[1].port == 18447
```

Comment:
When this fails, you can get a more complete view of the failure in the pytest output by doing something like `assert peer_infos == [UnresolvedPeerInfo(.....), UnresolvedPeerInfo(.....)]` and filling out the needed bits.  This also is easier if you end up switching to the sets as commented about above.